docs/source/features/post_screening.rst

```diff
@@ -16,7 +16,8 @@ any time during the screening process. To download your results:
 2. Start a new project, upload a dataset and select prior knowledge.
 3. Navigate to the Project Dashboard.
 4. Click the *download* icon (see screenshot below), or click on *Download Results*.
-5. You will be asked whether you want to save an excel or a csv file.
+5. You will be asked whether you want to save an Excel file, a CSV file, or a RIS file containing the
```

Comment:
This line renders an error in the documentation:

`Enumerated list ends without a blank line; unexpected unindent.`